src/app/models/field/dateField.ts

```diff
@@ -23,14 +26,15 @@ const createDateFieldSchema = () => {
         default: null,
       },
     },
-    invalidDaysOfTheWeek: {
+    validDays: {
       type: [
         {
           type: String,
           required: true,
         },
       ],
-      enum: [...Object.values(DaysOfTheWeek)],
+      enum: [...Object.values(ValidDaysOptions)],
+      default: Object.values(ValidDaysOptions),
```

Comment:
i don't think we should inverse the relationship. This means that all old date fields without the validDays prop (because the docs may not have been modified yet) will suddenly disallow all dates. We should stick with invalid-if-present model.